app/src/main/res/layout/preference.xml

```diff
@@ -0,0 +1,29 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Use `<merge>` tag for root view instead.

Reply:
I've never seen this, do you have an example?

Reply:
Use google for that.

Reply:
this https://developer.android.com/develop/ui/views/layout/improving-layouts/reusing-layouts#:~:text=The%20tag%20helps%20eliminate,view%20by%20extending%20a%20ViewGroup%20. ?

Reply:
good bro, I took a look, I used it here, as you said, but the result was not satisfactory at all. overlapping texts, wrong orientation, so, in my opinion, it is better to keep it as it is.

Reply:
You need to move attributes to Java class.

Reply:
> You need to move attributes to Java class.
> 
> `app/src/main/java/mod/trindadedev/ui/preferences/Preference.java`
> 
> ```java
> setOrientation(VERTICAL);
> // and so on...
> ```

ok, thanks, I'll go to sleep and then finish the requested changes

Reply:
Brother, tell me, honestly, why this is? I did what you asked here, and I didn't see any advantage in doing this.

Reply:
Same, LMAO.

Reply:
> Same, LMAO.

🤔